bigtable_automation/gcf/prod/base.yaml

```diff
@@ -2,7 +2,7 @@ projectID: "datcom-store"
 instance: "prophet-cache"
 cluster: "prophet-cache-c1"
 nodesHigh: "298"
-nodesLow: "20"
+nodesLow: "30"
```

Comment:
@Spaceenter so we want to change this one?

Reply:
Yes

Reply:
yes